docs/_extra/api-reference/hypothesis-v2.yaml

```diff
@@ -100,7 +100,7 @@ info:
     | 500         | Server Error  | An error occurred with our API |
 
 servers:
-  - url: https://api.hypothes.is/api
+  - url: https://hypothes.is/api
```

Comment:
Every change made to `hypothesis-v1.yaml` has to be duplicated in `hypothesis-v2.yaml` as well.